src/imitation/algorithms/adversarial/common.py

```diff
@@ -493,6 +499,10 @@ def _get_log_policy_act_prob(
             assert gen_algo_actor is not None
             # generate log_policy_act_prob from SAC actor.
             mean_actions, log_std, _ = gen_algo_actor.get_action_dist_params(obs_th)
+            assert isinstance(
+                gen_algo_actor.action_dist,
+                distributions.SquashedDiagGaussianDistribution,
+            )  # Note: this is just a hint to mypy
```

Comment:
IMO unnecessary comment, but no harm in keeping.

Reply:
Still seems redundant to me (ie, its status as a hint to mypy is obvious), but I don't have that strong of a preference for removing. If you do remove it here, I think you've got 3 equivalent comments in another file that you should remove too.